client/my-sites/plugins/plugin-meta/style.scss

```diff
@@ -128,11 +113,26 @@
 
 }
 
+.plugin-meta__actions .form-toggle__label {
+	display: flex;
+		justify-content: flex-end;
+
+	.form-toggle__switch {
+		float: none;
+		order: 2;
+	}
+}
+
 .plugin-meta__actions .plugin-action__label {
 	@include breakpoint( '<480px' ) {
+		flex-grow: 2;
 		color: $gray-dark;
 		font-size: 11px;
 	}
+	@include breakpoint( '<780px' ) {
```

Comment:
@johnHackworth Do you mind cleaning this up? This is not one of our supported breakpoint sizes so this will throw a warning. I'd recommend using a manual media query if you need this size, or picking the closest matching which would be 660px